src/libraries/Native/Unix/CMakeLists.txt

```diff
@@ -212,6 +212,26 @@ if(CLR_CMAKE_TARGET_UNIX)
     add_definitions(-DTARGET_UNIX)
 endif(CLR_CMAKE_TARGET_UNIX)
 
+# Ensure that awk is present
```

Comment:
This is needed to make version-scripts helpers to work in libs partition. 
We should consider switching libs to use the `configurecompiler.cmake`. Not in this PR though.

Reply:
https://github.com/dotnet/runtime/blob/90c647d293550e0226f7d163a62fdc1ad7688f0b/eng/native/configurecompiler.cmake#L636-L662
I think these lines belong in `eng/native/configuretools.cmake`, which is in use by libraries partition. That will prevent us from re-duplicating the code, which was previously de-duplicated. 🙂

Reply:
@am11  right, `configuretools.cmake` is a better place. Thanks!